src/server/game/Entities/Creature/Creature.h

```diff
@@ -310,8 +316,8 @@ class Creature : public Unit, public GridObject<Creature>, public MovableMapObje
     [[nodiscard]] bool hasInvolvedQuest(uint32 quest_id)  const override;
 
     bool isRegeneratingHealth() { return m_regenHealth; }
-    void SetRegeneratingHealth(bool c) { m_regenHealth = c; }
-    void SetRegeneratingPower(bool c) { m_regenPower = c; }
+    void SetRegeneratingHealth(bool creature) { m_regenHealth = creature; }
+    void SetRegeneratingPower(bool creature) { m_regenPower = creature; }
```

Comment:
creature doesn't really make sense here as you aren't passing a creature object. `bool on` would make more sense

Reply:
or enable, or something along those lines

Reply:
I didn't pay attention to the variable type. It's fix.